Examples/Cocoa/Sources/Objective_C/include/Pin.h

```diff
@@ -15,7 +15,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-typedef NS_ENUM(NSInteger, PinAttributionObjectsInternalType) {
+typedef NS_ENUM(unsigned char, PinAttributionObjectsInternalType) {
```

Comment:
How will this affect serialization? At Pinterest we use NSSecureCoding which might throw exceptions if the type has changed.

Reply:
It will still be written in / read out via -encodeIntegerForKey / -decodeIntegerForKey and cast to the enum type. My comment of how to get to the enum type name in ObjectiveCNSEncodingExtension.m below.

Reply:
Please the casts added.